notification/shoutrrr.go

```diff
@@ -91,6 +91,25 @@ func Send(ctx *api.Context, connectionName, shoutrrrURL, title, message string,
 	var params *types.Params
 	if properties != nil {
 		params = (*types.Params)(&allProps)
+	} else {
+		params = &types.Params{}
+	}
+
+	// NOTE: Until shoutrrr fixes the "UseHTML" props, we'll use the mailer package
+	if service == "smtp" {
+		parsedURL, err := url.Parse(shoutrrrURL)
```

Comment:
I think we need to handle the smtp user/pass/host/port specified in the url  - only smtp://system should default to the environment variables